src/plugins/content/.i18nrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "prefix": "content",
+  "paths": {
+    "content": "."
+  },
+  "translations": ["translations/ja-JP.json"]
```

Comment:
I am not sure what is the purpose of this config file, but it seems wrong that it specifies here a single ja-JP translation, 
I would understand if it specified all available translation (which it probably also shouldn't do because there could be more) or no translation key at all